partition_processor.go

```diff
@@ -69,7 +69,9 @@ func newPartitionProcessor(partition int32,
 	lookupTables map[string]*View,
 	consumer sarama.Consumer,
 	producer Producer,
-	tmgr TopicManager) *PartitionProcessor {
+	tmgr TopicManager,
+	backoff Backoff,
```

Comment:
couldn't we take that from the options instead of adding extra parameters?

Reply:
Then the function needs to be adopted since the builder in the options can return an error and we can't ensure the backoff creation works correctly.

Reply:
well good point, let's keep it then.